rhobs/alerting/data_plane/prometheus.eaas_alerts.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-eaas-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: quota-alerts
+      interval: 1m
+      rules:
+        - alert: HypershiftAWSQuotaAlmostFull
+          annotations:
+            summary: Environment as a Service is approaching the quota for concurrent HyperShift AWS clusters
+            description: >-
+              Environment as a Service has {{ $value | humanize }} concurrent HyperShift AWS clusters.
+              The maximum allowed is 100.
+            alert_team_handle: <!subteam^S07NDQV6A4D> # @konflux-vanguard
+            runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/eaas/hypershift-aws-quota-almost-full.md
+          expr: sum(platform:hypershift_hostedclusters:max{platform="AWS"}) >= 80
```

Comment:
Is that quota global to all our clusters?

Reply:
Updated.